index.js

```diff
@@ -28,9 +28,10 @@ function indentResolve(str, options) {
     return str;
 }
 
-module.exports = function(plugins, options) {
+module.exports = function(plugins, options, filterType) {
```

Comment:
At the time, I'd thought that doing so would be tricky to handle empty attr variations. However, I could do something like:
```js
var includesStandardType = options.filterType.test("text/css");
```